### PR TITLE
included spring framework for better run

### PR DESCRIPTION
**Pull request by Vinay-M-G**
included spring framework for better run
Added spring bean xml file
included respective annotations

### CODE REVIEW

File: src/main/java/com/screenshotToDocument/App.java [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
 
 public class App 
 {
     public static void main( String[] args )
-    {	
-    	SessionServiceImpl sessionService = new SessionServiceImpl();
+    {
+        ApplicationContext context = new ClassPathXmlApplicationContext("spring_beans.xml");
+        System.out.println(context.containsBean("utility"));
```
Comment (Vinay-dev1345) [inline src/main/java/com/screenshotToDocument/App.java]: remove this

**Review by Vinay-dev1345 — CHANGES_REQUESTED** [review]
remove unused imports

File: src/main/java/com/screenshotToDocument/App.java (hunk 1 above)
Comment (Vinay-M-G) [reply src/main/java/com/screenshotToDocument/App.java]: removed

**Review by Vinay-dev1345 — APPROVED** [review]
LGTM!!

**Pull request merged**